app/src/main/res/values/strings.xml

```diff
@@ -122,6 +122,8 @@
     <string name="snackbar_file_not_deleted">File can\'t be deleted.</string>
     <string name="snackbar_folder_not_created">Error on creating application folder.</string>
     <string name="pdf_merged">PDFs Merged</string>
+    <string name="excel_selected">Excel file selected: </string>
```

Comment:
You are showing this string on snackbar, so remove `:` from this string

Reply:
Sure